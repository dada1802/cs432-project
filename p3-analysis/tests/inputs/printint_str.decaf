def int main()
{
    print_int("buyue box");
    return 0;
}