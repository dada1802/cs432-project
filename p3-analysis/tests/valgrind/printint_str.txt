==700962== Memcheck, a memory error detector
==700962== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==700962== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==700962== Command: ../decaf inputs/printint_str.decaf
==700962== 
Type mismatch in parameter 0 of call to 'print_int': expected int but found str on line 3
==700962== 
==700962== HEAP SUMMARY:
==700962==     in use at exit: 0 bytes in 0 blocks
==700962==   total heap usage: 1,801 allocs, 1,801 frees, 215,368 bytes allocated
==700962== 
==700962== All heap blocks were freed -- no leaks are possible
==700962== 
==700962== For lists of detected and suppressed errors, rerun with: -s
==700962== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
