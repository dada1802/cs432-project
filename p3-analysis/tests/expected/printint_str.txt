Type mismatch in parameter 0 of call to 'print_int': expected int but found str on line 3
